config/webpack.config.dev.js

```diff
@@ -7,6 +7,7 @@ var InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
 var WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
 var getClientEnvironment = require('./env');
 var paths = require('./paths');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
The reason we extract on dev is we use this build in an integrated env with the other python apps (this includes SSR). JS only option causes SSR to crash as it requires the `window` object